src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml

```diff
@@ -182,6 +183,25 @@
                 <td>
                     <a class="review-name" asp-page="./Review" asp-route-id="@assembly.ReviewId">@assembly.DisplayName</a>
                 </td>
+                <td>
+                    @if (assembly.Revisions.Count > 0)
+                    {
+                        var lastRevision = assembly.Revisions.Last();
+                        if (lastRevision.Approvers.Count > 0)
+                        {
+                            <span>
+                                @foreach (var approver in lastRevision.Approvers)
+                                {
+                                    <label class="text-success">✔️@approver</label>
+                                }
+                            </span>
+                        }
+                        else
+                        {
+                            <span class="text-danger">Pending</span>
```

Comment:
Can you deploy to the staging, please? I worry this might cause the list page to become too busy/red.

Reply:
That's right. It's much read since most are not approved. I am thinking of changing this to Approved ? Yes or No. With no color coding.